config.example.yaml

```diff
@@ -128,14 +127,26 @@ templates:
             type: file
             options:
               paths:
-                /v1/std/html/{title}:
+                /v1/html/{title}:
                   get:
                     backend_request:
-                      uri: http://appservice.wmflabs.org/{domain}/v1/mobile/app/page/html/{title}
-                /v1/lite/{title}:
+                      uri: http://appservice.wmflabs.org/en.wikipedia.org/v1/page/mobile-html/{title}
```

Comment:
Duh, copy/paste fail. I've never been good at that. Thnx @Pchelolo for noticing!